lib/Core/KnockoutMarkdownBinding.js

```diff
@@ -10,6 +9,7 @@ var KnockoutMarkdownBinding = {
     allowUnsafeHtml : false,
 
     register : function(knockout) {
+        var allowUnsafeHtml = this.allowUnsafeHtml;
```

Comment:
This is a pretty minor point, but you should do `var that = this` here and then access `that.allowUnsafeHtml` later, instead of saving `this.allowUnsafeHtml`.  That way changing the value after registration will have an effect.